### DISCUSSION TITLE
Is there hot reload in previous versions + docker?

### DISCUSSION THREAD

**Discussion by dan-developer** [dotnet/dotnet-docker#3876] [Q&A]
I'm trying to develop using Jetbrains Rider and a web application in .net 3.1.

Just like other development (web) stacks when we apply a change, the application reloads (inside or outside of docker) and updates are immediately available.

Is this possible with .net + docker?

**Comment by richlander** [comment]
Hot reload is a .NET 6 feature, including both SDK and runtime changes, so I don't think so. I'm not an expert on the feature. The most straightforward thing to do is move to .NET 6.

**Reply by dan-developer** [reply]
Thanks a lot for the help.

I would like a sure confirmation before migrating to version 6.

I found it strange that this feature did not exist for so long.

**Reply by richlander** [reply]
Docs are here: https://docs.microsoft.com/aspnet/core/test/hot-reload?view=aspnetcore-6.0

You can take a few minutes and try it in a .NET 6 SDK container images with `dotnet watch`, with volume mounting. Check out our samples: https://github.com/dotnet/dotnet-docker/tree/main/samples.
